mozilla-release/mobile/android/base/java/org/mozilla/gecko/tabs/TabsPanelThumbnailView.java

```diff
@@ -58,19 +60,44 @@ public void setImageDrawable(Drawable drawable) {
         boolean resize = true;
 
         if (drawable == null) {
-            drawable = getResources().getDrawable(R.drawable.tab_panel_tab_background);
+            StateListDrawable states = new StateListDrawable();
+            states.addState(PRIVATE_STATE_SET, privateModeImageDrawable());
+            states.addState(EMPTY_STATE_SET, normalModeImageDrawable());
+            drawable = states;
             resize = false;
-            setScaleType(ScaleType.FIT_XY);
+            setScaleType(ScaleType.CENTER_CROP);
         }
 
         super.setImageDrawable(drawable, resize);
     }
 
-    /* Cliqz start */
+    private LayerDrawable normalModeImageDrawable() {
+        AppCompatDelegate.setCompatVectorFromResourcesEnabled(true);
+        Drawable backgroundLayer = ContextCompat.getDrawable(getContext(), R.color.tab_item_thumbnail_default_bg);
```

Comment:
Do not call `getContext()` every time you need a context in the same method. `final Context context = getContext();` and use replace `getContext()` with `context`.